lfs/gitscanner.go

```diff
@@ -74,9 +74,9 @@ func (s *GitScanner) RemoteForPush(r string) error {
 	return nil
 }
 
-// ScanLeftToRemote scans through all commits starting at the given ref that the
+// ScanRangeToRemote scans through all commits starting at the given ref that the
```

Comment:
Since there's a new parameter here (`right`) and we're now providing two refs, could you update this comment accordingly?